taxi/views.py

```diff
@@ -87,3 +88,34 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+    def get_success_url(self):
+        return reverse("taxi:driver-detail", kwargs={"pk": self.object.pk})
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+def add_driver_to_car(request, pk):
+    car = get_object_or_404(Car, id=pk)
+    car.drivers.add(request.user)
+    return redirect("taxi:car-detail", pk=car.id)
+
+
+def remove_driver_from_car(request, pk):
+    car = get_object_or_404(Car, id=pk)
+    car.drivers.remove(request.user)
```

Comment:
The `remove_driver_from_car` function currently removes the requesting user as a driver without any permission checks. Implement proper permission checks to ensure that only authorized users can remove drivers from a car.